s3/replication/manager/src/s3replicationmanager/config.py

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env python3
+
+#
+# Copyright (c) 2021 Seagate Technology LLC and/or its Affiliates
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# For any questions about this software or licensing,
+# please email opensource@seagate.com or cortx-questions@seagate.com.
+#
+
+# Replication manager config
+#
+# This file contains config class and
+# functions to apply config
+
+# import sys
+import os
+# import shutil
+import logging
+import yaml
+# import uuid
+# import glob
+import logging.handlers
+
+class Config:
+    """ configuration for manager """
+
+    rep_conf = {}
+    logfile_name = 'default.log'
+    logfile_size = 5242880  # default 5MB
+    rotation = 5
+    location = './'
+    host = '127.0.0.0'
+    port = 8080
+    configfile = os.path.join(os.path.dirname(__file__), '../config/config.yaml')
+
+    def __init__(self, configfile):
+        """config class constructor"""
+        self.logger = logging.getLogger('manager_proc')
+        self.logger.setLevel(logging.DEBUG)
+
+        if configfile is not None:
+            self.configfile = configfile
+
+        self.load_config()
+        self.set_config()
+
+        # Add the log message handler to the logger
+        formatter = logging.Formatter('%(asctime)s [%(levelname)s] [%(filename)s: %(lineno)d] %(message)s')
```

Comment:
All this code should move inside log module and we can just pass in the override read from config.

Reply:
Logging params are coming from input config.yaml. 
In this function, logger of manager/replicator(which is already present with default config) is populated with yaml's logging parameter. It is good to keep log.py with default configs and make config related changes all at one place???